helm/alfresco-content-services/charts/alfresco-sync-service/values.yaml

```diff
@@ -15,7 +15,6 @@ podSecurityContext:
   runAsNonRoot: true
   fsGroup: 1000
 syncservice:
-  enabled: true
```

Comment:
it's still used in the [deployment](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/charts/alfresco-sync-service/templates/deployment-syncservice.yaml#L1)